### PR TITLE
Modeling users

### PR DESCRIPTION
**Pull request by agalkiewicz**

### CODE REVIEW

File: test/fixtures/users.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (mmarcinkowski) [inline test/fixtures/users.yml]: Do u really need this comment and file ? :wink:

File: test/models/user_test.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+    assert_not @user.valid?
+  end
+
+  test "email should not be too long" do
+    @user.email = "a" * 244 + "@example.com"
+    assert_not @user.valid?
+  end
+
+  test "email validation should accept valid addresses" do
+    valid_addresses = %w[user@example.com USER@foo.COM A_US-ER@foo.bar.org
```
Comment (mmarcinkowski) [inline test/models/user_test.rb]: what do u think about something like that:
```
valid_addresses = 
  %w[user@example.com USER@foo.COM A_US-ER@foo.bar.org first.last@foo.jp alice+bob@baz.cn]
```

File: db/migrate/20150831224902_add_index_to_users_email.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddIndexToUsersEmail < ActiveRecord::Migration
+  def change
+    add_index :users, :email, unique: true
```
Comment (mmarcinkowski) [inline db/migrate/20150831224902_add_index_to_users_email.rb]: https://tomafro.net/2009/08/using-indexes-in-rails-index-your-associations

File: test/models/user_test.rb (hunk 2 above)
Comment (agalkiewicz) [inline test/models/user_test.rb]: It's more than 100 characters in one line. I prefer the first option.

Comment (mmarcinkowski) [inline test/models/user_test.rb]: kk

File: db/migrate/20150831224902_add_index_to_users_email.rb (hunk 3 above)
Comment (agalkiewicz) [inline db/migrate/20150831224902_add_index_to_users_email.rb]: Interesting, thanks.

File: test/models/user_test.rb (hunk 2 above)
Comment (agalkiewicz) [inline test/models/user_test.rb]: Changed.

File: test/fixtures/users.yml (hunk 1 above)
Comment (agalkiewicz) [inline test/fixtures/users.yml]: Deleted.

**Comment by mmarcinkowski** [comment]
:+1:

**Pull request merged**